cmake/PythonSetupAPIinBuild.cmake

```diff
@@ -21,4 +21,5 @@ configure_file( "${API_SOURCE_DIR}/api.py" "${API_TARGET_DIR}/api.py" )
 configure_file( "${API_SOURCE_DIR}/func.py" "${API_TARGET_DIR}/func.py" )
 configure_file( "${API_SOURCE_DIR}/runtime.py" "${API_TARGET_DIR}/runtime.py" )
 configure_file( "${API_SOURCE_DIR}/plugin.py" "${API_TARGET_DIR}/plugin.py" )
+configure_file( "${API_SOURCE_DIR}/state.py" "${API_TARGET_DIR}/state.py" )
```

Comment:
The new `state` object really needs to stand alone at the core of the EnergyPlus API, so I moved it to its own file.